package.json

```diff
@@ -34,8 +34,13 @@
     "@nuxtjs/dotenv": "^1.4.0",
     "@nuxtjs/pwa": "^3.0.0-beta.20",
     "@nuxtjs/svg": "^0.1.7",
+    "child_process": "^1.0.2",
+    "fs": "0.0.1-security",
+    "jsdom": "^16.2.2",
+    "net": "^1.0.2",
     "nuxt": "^2.12.2",
-    "sanitize.css": "^11.0.0"
+    "sanitize.css": "^11.0.0",
+    "tls": "0.0.1"
```

Comment:
How is this used?

Reply:
I tried figuring this out by removing this dependency from package.json then rebuilding and see what errors come up. After doing this there were no errors but after pushing those changes the test on the PR failed. I then readded those dependencies.

Reply:
Please take a look on test failures reported on PR and figure out why they occur on CI but not on your local computer. It will help us to understand which part is problematic. Thanks.